src/main/java/me/totalfreedom/totalfreedommod/command/Command_verified.java

```diff
@@ -0,0 +1,26 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.DepreciationAggregator;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.IMPOSTOR, source = SourceType.BOTH)
```

Comment:
@marcocorriero Not quite, as the terms imply, rank `IMPOSTOR` is available to non-op players.